plugins/maven/src/main/java/org/jetbrains/idea/maven/execution/MavenExternalParameters.java

```diff
@@ -527,6 +527,11 @@ private static void encodeCoreAndRunnerSettings(MavenGeneralSettings coreSetting
     if (coreSettings.getOutputLevel() == MavenExecutionOptions.LoggingLevel.DEBUG) {
       cmdList.add("--debug");
```

Comment:
Previous implementation seem to add exceptions as well (`-X` and `-e`).
Add it to debug level or create a separate trace level with all the output?
Or may be exception only on debug level and `--debug` on trace level?